react/store/createReturnRequest/components/PaymentMethods.tsx

```diff
@@ -175,6 +176,16 @@ export const PaymentMethods = ({ canRefundCard }: Props) => {
                 }
               />
             ) : null}
+            {bankDetailsError &&
+            refundPaymentData.iban &&
+            isValidIBANNumber(refundPaymentData.iban) !== 1 ? (
```

Comment:
Bring the validation `!==1` inside the return of `isValidIBANNumber`. This way, your function will "answer" the user calling it with the right answer -> true or false.